townhall/tests/test_opportunity.py

```diff
@@ -67,7 +67,7 @@ def test_update_opportunity(self):
             description = "Planting some flowers in the community center",
             location = "Richmond"
         )
-        townhall_services.OpportunityServices.update_opportunity(update_opportunity_data)
+        townhall_services.OpportunityServices.update_opportunity(id=3, update_opportunity_data=update_opportunity_data)
 
```

Comment:
I challenge in this PR as learning point to incorporate endpoint unit tests, for example `client.get` or `client.post` or `client.patch` . You can use this article as reference how to do this https://djangostars.com/blog/django-pytest-testing/

Note how they assert status code and the response. Please also make a new file called `test_endpoint.py` for these new set of tests (which inside will have 3 classes corresponding to each model that we have) . No rush on this changes I know its a lot, good luck and ping me if theres any qs

Reply:
@ansel-rangers11 Just committed an update if you could double check if it was done correctly or not. I used pytest to test the endpoints

Reply:
@RYeeAnn yes looks like youve got it! Ill approve once the merge conflicts have been resolved

Reply:
@ansel-rangers11 awesome thanks, just resolved the merge conflict